### PR TITLE
add new migrations phases info

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
Add new migration planning information to the migrations Intro doc, and convert to asciidoc

# Reasons
[A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.
](https://circleci.atlassian.net/browse/DOCSTEAM-191?atlOrigin=eyJpIjoiNzMyOWY5YjQ1OTllNDdjOGExM2RhMGI0NTIwYTAyYmMiLCJwIjoiaiJ9)
# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: https://circleci.com/docs/style/.

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [x] Break up walls of text by adding paragraph breaks.
- [x] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [x] Keep the title between 20 and 70 characters.
- [x] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [x] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [x] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [x] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

**Review by brianoh1979 — CHANGES_REQUESTED** [review]
Looks great! But I'd suggest the following minor changes:

(1) Underneath the list of 'Here are some basic resources to get you started' , the existing line "Enroll in the [migrationscourse](https://academy.circleci.com/arm-course?access_code=public-2021)
with CircleCI Academy to learn more about migrating to CircleCI." can be added preceded by the words 'You can also'.

I think that it is better here than at the very end.

(2) At the start of the "Migrating projects" section, the line beginning with "Any Jenkins project can be migrated...." should begin with "As an example, any Jenkins project can be migrated...." .

I would also change the title to "Project Migration Methodology" . 

(3) In the final section entitled "Next Steps", I would change the title to "Product-specific migration references"

**Comment by brianoh1979** [comment]
Okay, all looks good to me!

**Review by kbatuigas — CHANGES_REQUESTED** [review]
Just some minor suggestions!

File: jekyll/_cci2/migration-intro.adoc [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+For complex projects with a large-scale test suite migration and large workflows, the migration timeline for completing all phases would be roughly:
+
+-   1 project: ~5 weeks
+-   5 projects: ~4 months
+-   10 projects: ~9 months+
+
+[#assessment-phase]
+== Assessment phase
+
+The first phase in our journey is the Assess phase. This is where you’ll take stock of your current landscape, and understand the differences between CircleCI functionality and that of your current CI tool.
```
Comment (rosieyohannan) [inline jekyll/_cci2/migration-intro.adoc:58]:
```suggestion
Take stock of the current landscape, and understand the differences between CircleCI functionality and that of your current CI tool.
```

File: jekyll/_cci2/migration-intro.adoc [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+Part of the test migration is conducting User Acceptance Testing (UAT) so your end users can ensure that the project works on CircleCI as expected. Through this process you can uncover any issues that will impact your end users and help your teams prepare to work with CircleCI.
+
+[#communicate-your-plan]
+=== Communicate your plan
+
+With final timelines and owners in place, communicate the official plan to your organization. Things to include in your communications are:
+
+-   When the migration will occur
+-   Details of downtime users can expect
+-   Ask end-users to avoid changing anything during the transition
```
Comment (rosieyohannan) [inline jekyll/_cci2/migration-intro.adoc:120]:
```suggestion
-   Ask end users to avoid changing anything during the transition
```

**Review by brianoh1979 — APPROVED** [review]
This all looks good to me!

**Pull request merged**